internal/controllers/network/tests/create-full/00-assert.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Network
+metadata:
+  name: create-full-network
+status:
+  resource:
+    name: create-full-network
+    adminStateUp: true
+    description: "foo"
+    # Skipped until https://github.com/k-orc/openstack-resource-controller/issues/189 is fixed
+    #dnsDomain: "foo.com"
+    external: false
+    mtu: 1400
+    portSecurityEnabled: false
+    shared: false
+    tags:
```

Comment:
Are we not able to create a network with tags? CI complains about this [missing from the status](https://github.com/k-orc/openstack-resource-controller/actions/runs/13174101941/job/36769579541?pr=223#step:7:470).
If so, this would be a bug.